src/main/webapp/queue/ta.html

```diff
@@ -23,22 +23,60 @@
       });
     }
 
+    function notifyStudent(uEmail){
+      var params = window.location.search + "&uEmail=" + uEmail;
+      const request = new Request("/notify-student" + params, {method: "POST"});
+      document.getElementById('beingHelped').appendChild(createHelpedElem(uEmail));
+      
+      fetch(request);
+
+      getQueue();
+    }
+
+    function endHelp(uEmail){
+      var params = window.location.search + "&uEmail=" + uEmail;
+      const request = new Request("/end-help" + params, {method: "POST"});
+      document.getElementById('beingHelped').innerHTML = "";
+      
+      fetch(request);
+
+      getQueue();
+    }
+
     function createListElement(text) {
-      let notifyForm = document.createElement("form");
-      notifyForm.action = "/notify-student";
-      notifyForm.method = "post";
+      let notifyElem = document.createElement("div");
```

Comment:
You may want to consider `template` elements for creating list and help elements.